tests/cypress/e2e/for-teacher_page/class_page/class-survey.cy.js

```diff
@@ -0,0 +1,86 @@
+import {loginForTeacher} from '../../tools/login/login.js'
+
+const surveyView = body => body.find("#survey")
+const classView = body => body.find(".view_class")
+describe('Class Survey View', () => {
+  it('Can respond to 1 questions', () => {
+    
+    loginForTeacher();
+    cy.wait(500);
+
+    cy.get(".view_class").first().click()
+    cy.get("body").then(surveyView).then(survey => {
```

Comment:
This works, but is not really the Cypress way of doing it. You don't need to target the body and then find the survey, `cy.get` can select any element of the page, so doing `cy.get('#survey')` will do the job. Look at how we target elements on the other tests for some guidance

Reply:
The thing is, if a class doesn't contain students, there is no survey view, so `cy.get('#survey')` fails. So I was looking for a workaround and found this in other tests.

Reply:
Ohh I get it. CLASS1 in the database has some students, so maybe targetting that class is the way to go. Otherwise, you might create a new class, add some students and then run this test to guarantee that the survey will be shown.

Reply:
That's a good one! Will do that thanks.